osgp/platform/osgp-adapter-ws-smartmetering/src/main/resources/osgp-adapter-ws-smartmetering.properties

```diff
@@ -53,6 +53,10 @@ soap.message.printing.enabled=true
 
 application.name=OSGP_ADAPTER_WS_SMARTMETERING
 
+# Key to decrypt content coming from osgp
+osgp.rsa.private.key.resource=file:/etc/osp/smartmetering/keys/platform/osgp-protocol-adapter-rsa-private.key
```

Comment:
Now this confuses me a little.
I would not expect the GXF private key as a smartmetering key, as I think it would have broader scope.
I am not to sure if the protocol adapter in its name helps.
It looks as if this key already existed under this name, and is now only used in a new location in the code.
If that is indeed the case, we could leave it at this for this pull request, and have a look at the key naming and positioning of the GXF private RSA key at some later time.

Reply:
That's indeed the case.